chapters/packages.tex

```diff
@@ -278,34 +278,40 @@ \subsection{The within Clause}\label{the-within-clause}
 \section{External Resources}\label{external-resources}
 
 In order to reference external resources from documentation (such as links and images in html-text) and/or to reference images in the \lstinline!Bitmap! annotation (see \cref{bitmap}).
-Absolute URIs should be used, for example \filename{file:///} and the URI scheme \filename{modelica://} which can be used to retrieve resources associated with a package.
-According to the URI specification scheme names are case-insensitive, but the lower-case form should be used, that is \filename{Modelica://} is allowed but \filename{modelica://} is the recommended form.
+Absolute URIs should be used, for example \filename{file:///} and the URI scheme \filename{modelica:/} which can be used to retrieve resources associated with a package.
+According to the URI specification scheme names are case-insensitive, but the lower-case form should be used, that is \filename{Modelica:/} is allowed but \filename{modelica:/} is the recommended form.
 
 The Modelica-scheme\index{Modelica!URI scheme}\index{URI!Modelica} has the ability to reference a hierarchical structure of resources associated with packages.
 The same structure is used for all kind of resource references, independent of use (external file, image in documentation, bitmap in icon layer, and link to external file in the documentation), and regardless of the storage mechanism.
 
 Any Modelica-scheme URI containing a slash after the package-name is
-interpreted as a reference to a resource. The \emph{authority} portion of the
-URI is interpreted as a fully qualified package name and the \emph{path}
-portion of the URI is interpreted as the path (relative to the package)
+interpreted as a reference to a resource. The first \emph{segment} of the
+\emph{path} of the URI is interpreted as a fully qualified package name and the rest of the \emph{path}
+of the URI is interpreted as the path (relative to the package)
 of the resource. Each storage scheme can define its own interpretation
 of the path (but care should be taken when converting from one storage
 scheme or when restructuring packages that resource references resolve
 to the same resource). Any storage scheme should be constrained such
 that a resource with a given path should be unique for any package name
-that precedes it. The first part of the path shall not be the name of a
-class in the package given by the authority.
```

Comment:
Why was this removed rather than reformulated for the new form of Modelica-URI?

Reply:
Will re-add.